index.js

```diff
@@ -1,6 +1,59 @@
+    convertDagToTree : require('./lib/convertDagToTree.js'),
+    encodeImages : require('./lib/encodeImages.js'),
+    findAccessorMinMax : require('./lib/findAccessorMinMax.js'),
+    generateNormals : require('./lib/generateNormals.js'),
+    getAccessorByteStride : require('./lib/getAccessorByteStride.js'),
+    getBinaryGltf : require('./lib/getBinaryGltf.js'),
+    getUniqueId : require('./lib/getUniqueId.js'),
+    gltfPipeline : require('./lib/gltfPipeline.js'),
+    loadGltfUris : require('./lib/loadGltfUris.js'),
+    mergeBuffers : require('./lib/mergeBuffers.js'),
+    mergeDuplicateAccessors : require('./lib/mergeDuplicateAccessors.js'),
+    mergeDuplicateVertices : require('./lib/mergeDuplicateVertices.js'),
+    NodeHelpers : require('./lib/NodeHelpers.js'),
+    numberOfComponentsForType : require('./lib/numberOfComponentsForType.js'),
+    octEncodeNormals : require('./lib/octEncodeNormals.js'),
+    packArray : require('./lib/packArray.js'),
+    parseBinaryGltf : require('./lib/parseBinaryGltf.js'),
+    quantizeAttributes : require('./lib/quantizeAttributes.js'),
+    readAccessor : require('./lib/readAccessor.js'),
+    readBufferComponentType : require('./lib/readBufferComponentType.js'),
+    readGltf : require('./lib/readGltf.js'),
+    removeDuplicatePrimitives : require('./lib/removeDuplicatePrimitives.js'),
+    removePipelineExtras : require('./lib/removePipelineExtras.js'),
+    removeUnused : require('./lib/removeUnused.js'),
+    removeUnusedAccessors : require('./lib/removeUnusedAccessors.js'),
+    removeUnusedBuffers : require('./lib/removeUnusedBuffers.js'),
+    removeUnusedBufferViews : require('./lib/removeUnusedBufferViews.js'),
+    removeUnusedCameras : require('./lib/removeUnusedCameras.js'),
+    removeUnusedImages : require('./lib/removeUnusedImages.js'),
+    removeUnusedMaterials : require('./lib/removeUnusedMaterials.js'),
+    removeUnusedMeshes : require('./lib/removeUnusedMeshes.js'),
+    removeUnusedNodes : require('./lib/removeUnusedNodes.js'),
+    removeUnusedPrimitiveAttributes : require('./lib/removeUnusedPrimitiveAttributes.js'),
+    removeUnusedPrograms : require('./lib/removeUnusedPrograms.js'),
+    removeUnusedSamplers : require('./lib/removeUnusedSamplers.js'),
+    removeUnusedShaders : require('./lib/removeUnusedShaders.js'),
+    removeUnusedSkins : require('./lib/removeUnusedSkins.js'),
+    removeUnusedTechniques : require('./lib/removeUnusedTechniques.js'),
+    removeUnusedTextures : require('./lib/removeUnusedTextures.js'),
+    removeUnusedVertices : require('./lib/removeUnusedVertices.js'),
+    uninterleaveAndPackBuffers : require('./lib/uninterleaveAndPackBuffers.js'),
+    writeAccessor : require('./lib/writeAccessor.js'),
+    writeBinaryGltf : require('./lib/writeBinaryGltf.js'),
+    writeBufferComponentType : require('./lib/writeBufferComponentType.js'),
+    writeGltf : require('./lib/writeGltf.js')
+};
```

Comment:
The main problem with exposing a lot of these functions is that they may rely on previous steps, like `addPipelineExtras`. Either we need to handle that in the functions themselves, or expect users to call things like `addPipelineExtras`, `loadGltfUris`, etc on their own.